src/MidiConvert.d.ts

```diff
@@ -48,6 +48,7 @@ export interface MIDI {
 export function parse(raw: ArrayBuffer): MIDI;
 export function load(url: string, data?: any, method?: 'GET'|'POST'): Promise<MIDI>;
 export function create(): MIDI;
+export function fromJSON(json: object): MIDI;
```

Comment:
I wonder if `Partial<MIDI>` is a more accurate type for `json`.  `json` should have the same shape as `MIDI`, just without the methods, right?